OneCommunityGlobal/HighestGoodNetworkApp
# Aaryaneil - Added test cases to Project Table component.

# Description
Added unit tests to check for dark mode functionality and other test cases to Project Table component

## Main changes explained:
Added test cases to check the following:
1. renders the table with dark mode styles when darkMode is true
2. renders the table with light mode styles when darkMode is false
3. renders an empty table when no projects are provided
4. renders with correct table header labels

## How to test:
1. check into current branch
2. do `npm install` and `npm test ProjectTable.test.js `

## Screenshots or videos of changes:
<img width="827" alt="Screenshot 2024-06-14 at 12 29 25 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/39597927/74e95368-c422-4136-a0a5-4c5e22937303">
